modules/authorization/src/controllers/relations.controller.ts

```diff
@@ -2,14 +2,25 @@ import ConduitGrpcSdk from '@conduitplatform/grpc-sdk';
 import { checkRelation, computeRelationTuple } from '../utils';
 import { Relationship, ResourceDefinition } from '../models';
 import { IndexController } from './index.controller';
+import { Queue } from 'bullmq';
+import { randomUUID } from 'crypto';
 
 export class RelationsController {
   private static _instance: RelationsController;
+  private queue: Queue;
 
   private constructor(
     private readonly grpcSdk: ConduitGrpcSdk,
     private readonly indexController: IndexController,
-  ) {}
+  ) {
+    this.queue = new Queue('indexes', {
+      connection: {
```

Comment:
Redis details are already available in the `grpc-sdk` instance.
Getting them from the envs is redundant and doesn't cover all supported auth combos.